schema-statistics/columns-hot-updates.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Column statistics and HOT updates'
+backlink_href: /docs/schema-statistics
+backlink_title: 'Tracking Postgres schema statistics in pganalyze'
+---
+
+![](table-columns.png)
+
+On the "Columns" tab of each table you can find all columns that are defined on the table, as well as the planner statistics associated with the column (collected by ANALYZE):
+
+* **Null %**: Fraction of the rows in this table that have a NULL value in this column.
+* **Avg. Size**: Average size, in bytes, of the values in this column across all the rows in this table (excluding NULLs).
+* **Num. Distinct**: Estimated number of distinct values in this column. If followed by an asterisk (*), the number of values is expected to grow as the table grows (e.g., for a unique value like a primary key).
+
+In case of partitioned tables you will additionally see "Full NULL %", "Full Avg Size" and a second "Num. Distinct" data point. This represents the data that Postgres collects as an aggregate information across all partitions, as compared to just the partitioned table itself (in case of inheritance-based partitioning).
```

Comment:
```suggestion
In case of partitioned tables you will additionally see "Full NULL %", "Full Avg. Size" and a second "Num. Distinct" data point. This represents the data that Postgres collects as aggregate information across all partitions, as compared to just the partitioned table itself (in case of inheritance-based partitioning).
```

Also, this is for inheritance, not partitioning right? The most common usage is probably still partitioning, but I think we should focus on the underlying mechanism, since that's what the stats refer to. So maybe:

```suggestion
In case of tables with inheritance children (e.g., if using the older inheritance-based partitioning mechanism), you will additionally see "Full NULL %", "Full Avg. Size" and a second "Num. Distinct" data point. This represents the data that Postgres collects as aggregate information across all children (i.e., individual partitions), as compared to just the table itself.
```

We could maybe even add a "Note that this not apply to the new declarative partitioning mechanism (since the parent partitioned table is always empty). Check the [Postgres documentation on partitioning](https://www.postgresql.org/docs/current/ddl-partitioning.html) for details."

Reply:
Cool, will reword per your second suggestion. I'll leave out your last suggestion though, since I think we currently always display both statistics datapoints (i.e. top-level table and the inherited stats), even for declaratively partitioned tables.